webui/src/lib/api/index.js

```diff
@@ -764,9 +794,14 @@ class Config {
         }
     }
     async getLakeFSVersion() {
-        const response = await apiRequest('/config/version', {
-            method: 'GET',
-        });
+        try {
+            const response = await apiRequest('/config/version', {
```

Comment:
Think it is working because the final code uses 'var' - but we declare 'response' inside the scope and reference it outside the scope.